src/style.css

```diff
@@ -1 +1,82 @@
 /* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  background: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+  top: 56px;
+  left: 0;
+  right: 0;
+}
+
+.nav__link {
+  padding: 23px 0;
+  font-family: inherit;
+  margin-right: 20px;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  cursor: pointer;
+  height: 60px;
```

Comment:
height doesn't work for tag `a`
`a` is an inline element and hight doesn't work for in inline elements
you can remove it